mobile/src/components/course/components/course-modules/course-modules.tsx

```diff
@@ -1,31 +1,38 @@
 import React, { FC } from 'react';
 
-import { Stack, Text, View } from '~/components/common/common';
-import { useAppSelector } from '~/hooks/hooks';
+import { CourseModulesGetAllItemResponseDto } from '~/common/types/types';
+import { Spinner, Stack, Text, View } from '~/components/common/common';
 
 import { Module } from './components/module/module';
 import { styles } from './styles';
 
-const CourseModules: FC = () => {
-  const { courseModules } = useAppSelector((state) => state.courseModules);
+type Props = {
+  courseModules: CourseModulesGetAllItemResponseDto[];
+  isLoading: boolean;
+};
 
+const CourseModules: FC<Props> = ({ courseModules, isLoading }) => {
   if (!courseModules.length) {
     return <></>;
   }
 
   return (
     <View>
       <Text style={styles.title}>Course Content</Text>
-      <Stack space={15}>
-        {courseModules.map((module, index) => (
-          <Module
-            key={module.id}
-            index={index}
-            title={module.title}
-            description={module.description}
-          />
-        ))}
-      </Stack>
+      {isLoading ? (
+        <Spinner />
+      ) : (
```

Comment:
Move this outside 
```suggestion
     if(isLoading) {
       return <Spinner />
     }
```

Reply:
I decided to do like this, to show header 'Course modules'. You think it's better to show it with loaded modules?

Reply:
Oh, ok, that let's leave it as it is